submissions/mxmgny/a-tiny-js-world/index.js

```diff
@@ -0,0 +1,43 @@
+let cat = {
+    appearance: "cat",
+    name: "Garfield",
+    legs: 4,
+    hands: 0,
+    gender: "male",
+    shout: "I love lazagna!",
+    friends:['Selina', 'Jeremy']
+};
+let cat_woman = {
+    appearance:"cat-woman",
+    name:"Selina",
+    legs:2,
+    hands:2,
+    gender: "female",
+    shout:cat['shout'],
+    friends: [cat['name'],'Sansa']
+};
+let dog = {
+    appearance:"dog",
+    name: "Jerry",
+    gender: "male",
+    legs: 4,
+    hands: 0,
+    shout: "Woof!",
+    friends: ["Sansa","Snoop Dog", "Balto", "Goofy"]
+};
+let human = {
+    appearance:"human",
+    name:"Sansa",
+    gender:"female",
+    legs:2,
+    hands:2,
+    shout:"Winter is coming",
+    friends:[dog['name'], "Robb", "Arya", "Bran", "Rickon"]
+};
+
+let arr = [cat, dog, human,cat_woman];
```

Comment:
In this case, it is not hard to understand what for is that variable. But even if you work at a small project - take a good habit to give the most successful name to a variable, constant, ...etc. Perhaps, we could name it `Inhabitants` instead of `arr`? However, If you ask yourself: "this is an array of what?" - you may choose a name at your discretion.

Reply:
Done